src/smartcontracts/index.ts

```diff
@@ -8,17 +8,19 @@ export * from "./argSerializer";
 export * from "./code";
 export * from "./codec";
 export * from "./codeMetadata";
-export * from "./defaultRunner";
+export * from "./smartContractController";
 export * from "./function";
 export * from "./interaction";
+export * from "./interactionChecker";
 export * from "./interface";
 export * from "./nativeSerializer";
 export * from "./query";
 export * from "./queryResponse";
+export * from "./resultsParser";
 export * from "./returnCode";
 export * from "./smartContract";
 export * from "./smartContractResults";
-export * from "./strictChecker";
+export * from "./interactionChecker";
```

Comment:
Duplicate import? See line 14.

Reply:
Fixed. Yes, it was duplicated by mistake :)